content/docs/capabilities/high-availability.mdx

```diff
@@ -122,20 +134,25 @@ Multiple replicas of Databroker or all-in-one service are only supported with [e
 
 :::
 
-## Example
+## Example: Split-Service Mode
 
-The following setup would demonstrate a minimum configuration for the split-service mode, using docker compose on a local host.
+The following configuration demonstrates a minimum configuration for the split-service mode, using Docker Compose on a local host.
 
-This guide intentionally omits provisioning certificates for internal Pomerium interaction for simplicity.
+This guide intentionally omits provisioning certificates for internal Pomerium interactions for simplicity.
```

Comment:
What are "internal Pomerium interactions"? Does this refer to service-to-service communication? We might want to rephrase this to clarify.